OrderCloud.Integrations.Payment.CardConnect/CardConnectClient.cs

```diff
@@ -93,6 +94,98 @@ public static async Task<CardConnectFundReversalResponse> VoidPreviousAuthorizat
 			});
 		}
 
+		/// <summary>
+		/// https://developer.cardpointe.com/cardconnect-api#get-profile-request
+		/// </summary>
+		public static async Task<List<CardConnectProfile>> GetSavedCardsAsync(string profileid, string merchid, CardConnectConfig config)
```

Comment:
In the CardConnectClient class it is expected to use CardConnect's terminology for things. So this method name might be more like `GetProfileAsync()`.